packages/server/lib/controllers/connection.controller.ts

```diff
@@ -158,11 +160,12 @@ class ConnectionController {
 
             const providerName = integration.provider;
 
-            if (account.is_capped && provider_config_key) {
+            if (plan && plan.connection_with_scripts_max && provider_config_key) {
```

Comment:
why not `plan?.connection_with_scripts_max` btw?

Reply:
Changed